plasmapy/classes/plasma.py

```diff
@@ -1,14 +1,15 @@
 """
 plasmapy.classes.plasma
-===============
+=======================
 
 Defines the core Plasma class used by PlasmaPy to represent plasma properties.
 """
 
 import numpy as np
 import astropy.units as u
-
-mu0 = np.pi * 4.0e-7 * (u.newton / (u.amp**2))
+from astropy.utils.console import ProgressBar
```

Comment:
Right, because **of course** astropy would have a built-in progress bar for all things astronomical.